core/meta/src/TClass.cxx

```diff
@@ -4002,6 +4002,11 @@ void TClass::GetMissingDictionaries(THashTable& result, bool recurse)
       return;
    }
 
+   if (strncmp(fName, "unique_ptr<", 11) == 0 || strncmp(fName, "array<", 6) == 0) {
```

Comment:
And `tuple<`?

Reply:
humm .. good point ...

Reply:
added

Reply:
Those magic strings should probably be moved in a central place as we have them in various places already.